python-sdk/tests/databases/test_bigquery.py

```diff
@@ -155,6 +155,95 @@ def test_load_pandas_dataframe_to_table(database_table_fixture):
     assert rows[1] == (2,)
 
 
+@pytest.mark.integration
+@pytest.mark.parametrize(
+    "database_table_fixture",
+    [
+        {"database": Database.BIGQUERY},
+    ],
+    indirect=True,
+    ids=["bigquery"],
+)
+def test_load_pandas_dataframe_to_table_with_append(database_table_fixture):
```

Comment:
Can we not parameterise the test for all databases instead of writing the same test multiple times across databases?